packages/editor/src/components/post-format/index.js

```diff
@@ -38,6 +38,20 @@ export const POST_FORMATS = [
 	return 0;
 } );
 
+/**
+ * `PostFormat` is a React component that provides a dropdown selector for post formats.
+ * It fetches the current post format, suggested post format, and supported formats from the store.
+ * It filters the post formats based on the supported formats and the current post format.
+ * It also provides a suggestion for the post format if it's different from the current post format.
+ *
+ * @component
+ * @example
+ * ```jsx
+ * <PostFormat />
+ * ```
+ *
+ * @returns {React.Element} The rendered PostFormat component.
```

Comment:
This seems to cause a linter warning. I saw we usually use `@return {JSX.Element}` or ` @return {Component}`.